src/views/partials/navbar.hbs

```diff
@@ -0,0 +1,12 @@
+<nav class="dt w-100 bg-navbar center" id="navbar">
+  <div class="dtc w2 v-mid ">
+      <a class="f1  b no-underline dark-gray mh3" href="/">Ain</a>
+  </div>
+  <div class="dtc v-mid tr">
```

Comment:
In the first div no i just removed it, but in the second yes to keep the links(tabs) in the center of the div